d/config.yaml

```diff
@@ -1,5 +1,5 @@
 language:
-  version: 2.096
+  version: 1.32.2
```

Comment:
?

ℹ️ Not sure how to determine language version

Reply:
D has several compilers.
DMD has versions 2.XXX - now it is 2.104 https://dlang.org/download.html#dmd
LDC (based on LLVM) has different versions. The latest is 1.32.2 https://github.com/ldc-developers/ldc/releases

Reply:
If using `ldc` package of debian, how can we be sure of version number (of the language) ?

Reply:
I just saw that C uses Debian as well. So you are right - we can rely only on version from the package. Could we use it as temporary decision? While the updated versions of official (or at least community-driven) docker images are under preparation?